packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -170,7 +170,7 @@ class Directionality extends InheritedWidget {
 ///
 /// ```dart
 /// Image.network(
-///   'https://github.com/flutter/flutter_gallery_assets/raw/master/lib/products/backpack.png',
```

Comment:
I'd rather not redirect this to the website: it's structure can change, and then this link will be broken. Can you use an image that's in the assets-for-api-docs repo instead, like maybe [this one](
https://github.com/flutter/assets-for-api-docs/blob/master/packages/diagrams/assets/blend_mode_destination.jpeg?raw=true)?

Also, what's the reason for this change?

Reply:
Sure, done.